PayloadPkg/FirmwareUpdate/GetCapsuleImage.c

```diff
@@ -271,7 +271,7 @@ LoadCapsuleImage (
   }
 
   FsHandle = NULL;
-  Status = InitFileSystem (CapsuleInfo->SwPart, EnumFileSystemTypeFat, HwPartHandle, &FsHandle);
+  Status = InitFileSystem (CapsuleInfo->SwPart, EnumFileSystemTypeAuto, HwPartHandle, &FsHandle);
```

Comment:
should we use CapsuleInfo->FsType instead of hardcode?

Reply:
The patch now reworked to fix the real bug in FileSystem library.  Will cleanup and test API consumer code in follow-up changes.